server/src/main/java/org/elasticsearch/cluster/coordination/StableMasterHealthIndicatorService.java

```diff
@@ -83,7 +84,7 @@ public String name() {
     }
 
     @Override
-    public HealthIndicatorResult calculate(boolean explain) {
+    public HealthIndicatorResult calculate(boolean explain, HealthInfo healthInfo) {
```

Comment:
Shall we document that the `healthInfo` is not used here or give it a name that shows this? _(I miss the anonymous variable option in other languages)_

Reply:
Okay, I do not feel strongly about it.